app/models/asset.rb

```diff
@@ -448,6 +449,10 @@ def editable_image?
     !locked? && %r{^image/#{Regexp.union(Constants::WHITELISTED_IMAGE_TYPES_EDITABLE)}} =~ file.content_type
   end
 
+  def generate_base64(style)
```

Comment:
Why are we even passing style here?

Reply:
Currently we use it only in one place, but in future we can use it same as avatars, with multiple styles.